tslint.json

```diff
@@ -8,39 +8,20 @@
     "no-unused-expression": [true, "allow-fast-null-checks"],
     "ordered-imports": [false],
     "interface-name": [false],
-    "variable-name": [false], // Ben cries
+    "variable-name": [false],
     "member-access": [false],
-    "ban-types": [false],
     "array-type": [false],
-    "no-angle-bracket-type-assertion": [false],
-    "no-shadowed-variable": [false],
     "no-empty-interface": [false],
-    "interface-over-type-literal": [false],
     "member-ordering": [false],
-    "only-arrow-functions": [false],
     "callable-types": [false],
-    "prefer-const": [false],
     "object-literal-sort-keys": [false],
     "no-this-assignment": [false],
-    "one-variable-per-declaration": [false],
     "no-conditional-assignment": [false],
-    "no-unnecessary-initializer": [false],
-    "max-classes-per-file": [true, 10],
+    "max-classes-per-file": [false, 1000],
+    // TODO: unified signatures not be disabled, but it will
+    // be a lot of work to verify the fixes don't mess something up.
     "unified-signatures": [false],
```

Comment:
thought: TBH, I don't agree with this rule anyway. Sometimes separate signatures are clearer.